API/Import.php

```diff
@@ -119,17 +156,10 @@ public function importContainerVersion($exportedContainerVersion, $idSite, $idCo
     {
         $this->checkImportContainerIsPossible($exportedContainerVersion, $idSite, $idContainer);
 
-        foreach ($this->tags->getContainerTags($idSite, $idContainerVersion) as $tag) {
-            $this->tags->deleteContainerTag($idSite, $idContainerVersion, $tag['idtag']);
-        }
-
-        foreach ($this->triggers->getContainerTriggers($idSite, $idContainerVersion) as $trigger) {
-            $this->triggers->deleteContainerTrigger($idSite, $idContainerVersion, $trigger['idtrigger']);
-        }
-
-        foreach ($this->variables->getContainerVariables($idSite, $idContainerVersion) as $variable) {
-            $this->variables->deleteContainerVariable($idSite, $idContainerVersion, $variable['idvariable']);
-        }
+        $now = Date::now()->getDatetime();
+        $this->tagsDao->deleteContainerTags($idSite, $idContainerVersion, $now);
```

Comment:
@ulcuber Can you create a `deleteContainerTags()` in `Model/Tag.php` and call the Model instead of `dao`